packages/fuzzer/CMakeLists.txt

```diff
@@ -90,6 +119,8 @@ ExternalProject_Add(
              -DCOMPILER_RT_USE_LIBCXX=OFF
              # Use the same build type as the parent project.
              -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+             -DCOMPILER_RT_DEFAULT_TARGET_TRIPLE=${COMPILER_RT_DEFAULT_TARGET_TRIPLE}
+             -DLLVM_CMAKE_DIR=<SOURCE_DIR>/llvm/cmake/modules
```

Comment:
I don't seem to need these 2 lines for a successful build. Was that an issue on other systems?

Reply:
It builds now on my openSuse :smile:

Reply:
Yes, the first line is necessary for systems that, for whatever reason, don't populate `LLVM_TARGET_TRIPLE`. The second is necessary because the build was breaking on Windows (in CI at least) without it. I think it may be because windows doesn't have an llvm installation for `find_package` to find and so I had to tell it to look for llvm in the compiler-rt source. Macs would already have an install of it and ubuntu might if it reused the machine from the linting job though I didn't look into it very hard.

Reply:
Since it's required by some systems and doesn't break others, that's fine. Changes look good to me then, as it works fine on my machine as well. Also, it works locally for @oetr  as well.